src/helpers/helpers.core.js

```diff
@@ -32,13 +32,21 @@ var helpers = {
 	},
 
 	/**
-	 * Returns true if `value` is an array, else returns false.
+	 * Returns true if `value` is an array (including typed arrays), else returns false.
 	 * @param {*} value - The value to test.
 	 * @returns {Boolean}
 	 * @function
 	 */
-	isArray: Array.isArray ? Array.isArray : function(value) {
-		return Object.prototype.toString.call(value) === '[object Array]';
+	isArray: function(value) {
+		var type = Object.prototype.toString.call(value);
+		// Typed array
+		if (type.substr(0, 7) === '[object' && type.substr(-6) === 'Array]') {
+			return true;
+		}
+		if (Array.isArray) {
+			return Array.isArray(value);
+		}
+		return type === '[object Array]';
```

Comment:
I think this method (`isArray`) is called mostly with untyped arrays so it may be better to first check if the given value is a basic array:

```javascript
isArray: function(value) {
    if (Array.isArray && Array.isArray(value)) {
        return true;
    }
    var type = Object.prototype.toString.call(value);
    if (type.substr(0, 7) === '[object' && type.substr(-6) === 'Array]') {
        return true;
    } 
    return false;
}
```

Reply:
Would putting the variable at top help minimization (in this case)?
```
isArray: function(value) {
    var type;
    if (Array.isArray && Array.isArray(value)) {
        return true;
    }
    type = Object.prototype.toString.call(value);
    if (type.substr(0, 7) === '[object' && type.substr(-6) === 'Array]') {
        return true;
    } 
    return false;
}
```

Reply:
It could in this specific case but since there is only one variable (no other vars to collapse), I think it's better to keep it next to its initialization.

Reply:
Also Array.isArray returns false for typed array, so this order doesn't work as is.